.github/workflows/github_actions.yml

```diff
@@ -27,12 +27,12 @@ jobs:
         SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
     - name: Code Coverage
       run: bundle exec fastlane coverage
-    - name: Lint
-      run: bundle exec fastlane lint
-    - name: Setup sonarqube
-      uses: warchant/setup-sonar-scanner@v8
-    - name: Send to Sonarcloud
-      run: bundle exec fastlane sonarqube
-      env: 
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        SONAR_TOKEN: ${{ secrets.SONARCLOUD_KEY }}
\ No newline at end of file
+    # - name: Lint
```

Comment:
Can we just add a comment here explaining why this is commented, similar to https://github.com/ionic-team/ion-android-filetransfer/blob/0b0908ebba27f155f97793bebe8a1ede939c26e5/.github/workflows/continuous_integration.yml#L34 ?

Reply:
Also, does Lint need to be disabled? Can we enable it?

Reply:
The Lint job fails because SwiftLint is not available. We need to spend some time fixing and improving the whole process.

Reply:
Ok! Fine by me then